components/p3a/p3a_message.h

```diff
@@ -38,7 +38,8 @@ class MessageMetainfo {
   base::Time date_of_install;
   base::Time date_of_survey;
   int woi;  // Week of install. Remove this occasionally and extract from above.
-  std::string country_code;
+  std::string country_code_from_timezone;
+  std::string country_code_from_locale;
```

Comment:
Class fields are supposed to be either `private` or `protected`, so we shouldn't be mixing up private and public fields in a class, like we are doing in this case.

This class used to be a `struct`, however it got converted into a `class` in previous related work[2], and access to the members never was corrected.

cc @DJAndries @iefremov 

[1] https://google.github.io/styleguide/cppguide.html#Access_Control
[2] https://github.com/brave/brave-core/commit/421f611cfc26b078d5732ae232665550c1394d6c